src/plugins/dashboard/public/application/dashboard_router.tsx

```diff
@@ -260,6 +260,7 @@ export async function mountApp({
   }
   render(app, element);
   return () => {
+    dataStart.search.session.clear();
```

Comment:
Not sure if it's a problem, but when editing a visualization from a dashboard, the session is cleared twice